Containerfile.compute_worker_podman

```diff
@@ -7,56 +7,45 @@ RUN dnf -y update && \
+    echo -e "worker:1:999\nworker:1001:64535" > /etc/subgid;
 
-# Setup user 
-RUN useradd worker; \
-echo -e "worker:1:999\nworker:1001:64535" > /etc/subuid; \
-echo -e "worker:1:999\nworker:1001:64535" > /etc/subgid;
-
-# Copy over the podman container configuration
+# Copy over the podman container and storage configurations
 COPY podman/containers.conf /etc/containers/containers.conf
-COPY podman/worker-containers.conf /home/worker/.config/containers/containers.conf
-
-# Copy over the podman storage configuration
-COPY podman/worker-storage.conf /home/worker/.config/containers/storage.conf
-
-RUN mkdir -p /home/worker/.local/share/containers && \
-    chown worker:worker -R /home/worker && \
-    chmod 644 /etc/containers/containers.conf
+COPY podman/worker-*.conf /home/worker/.config/containers/
 
 # Copy & modify the defaults to provide reference if runtime changes needed.
 # Changes here are required for running with fuse-overlay storage inside container.
-RUN sed -e 's|^#mount_program|mount_program|g' \
-           -e '/additionalimage.*/a "/var/lib/shared",' \
-           -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' \
-           /usr/share/containers/storage.conf \
-           > /etc/containers/storage.conf
+RUN mkdir -p /home/worker/.local/share/containers && \
+    chown worker:worker -R /home/worker && \
+    chmod 644 /etc/containers/containers.conf && \
+    sed -e 's|^#mount_program|mount_program|g' \
+        -e '/additionalimage.*/a "/var/lib/shared",' \
+        -e 's|^mountopt[[:space:]]*=.*$|mountopt = "nodev,fsync=0"|g' \
+        /usr/share/containers/storage.conf > /etc/containers/storage.conf
 
 # Add volume for containers
 VOLUME /home/worker/.local/share/containers
 
-# Create directory for tmp space
-RUN mkdir /codabench && \
-    chown worker:worker /codabench
-
-# Set up podman registry for dockerhub
-RUN echo -e "[registries.search]\nregistries = ['docker.io']\n" > /etc/containers/registries.conf
-
 # This makes output not buffer and return immediately, nice for seeing results in stdout
-ENV PYTHONUNBUFFERED 1
-ENV CONTAINER_ENGINE_EXECUTABLE podman
-
-# Get pip for 3.8 
-RUN python3.8 -m ensurepip --upgrade
+ENV PYTHONUNBUFFERED=1 CONTAINER_ENGINE_EXECUTABLE=podman
 
 WORKDIR /home/worker/compute_worker
 
 ADD compute_worker/ /home/worker/compute_worker
 
-RUN chown worker:worker -R /home/worker/compute_worker
-
-RUN pip3.8 install -r /home/worker/compute_worker/compute_worker_requirements.txt
+# Create directory for tmp space
+# Set up podman registry for dockerhub
+RUN mkdir /codabench && \
+    chown worker:worker /codabench && \
+    echo -e "[registries.search]\nregistries = ['docker.io']\n" > /etc/containers/registries.conf && \
+    # Get pip for 3.8 
+    python3.8 -m ensurepip --upgrade && \
+    chown worker:worker -R /home/worker/compute_worker && \
+    pip3.8 install -r /home/worker/compute_worker/compute_worker_requirements.txt
 
 CMD celery -A compute_worker worker \
```

Comment:
@dtuantran How much space does this save? Just wondering if it's worth it given the loss of readability and ease of development, by having multiple steps.

Reply:
It doesn't save much space, sorry that I push it too quick, I've reverted back this file.